INSTALL.md

```diff
@@ -0,0 +1,62 @@
+## Installation instructions
+**Note: GCconnex does not work with PHP7 - GCconnex ne fonctionne pas avec PHP7**
+
+### Ubuntu 14.04
+#### Install Git, Apache, MySQL, PHP and libs
+    sudo aptitude install git apache2 mysql-server php5 libapache2-mod-php php5-mysql php5-gd
+When prompted, enter a root password for MySQL.
+
+#### Fork and Clone GCConnex Github Repo
+    git clone https://github.com/tbs-sct/gcconnex.git
+    git checkout gcconnex
+
+#### Create data directory
+    mkdir gcconnex_data
+
+#### Set permissions
+    chmod 777 gcconnex
+    chmod 777 gcconnex/engine
+    chmod 777 gcconnex_data
+A better option for the data folder would be to change the owner to the apache user.
```

Comment:
Since this (chown) is the recommended  option, we should at least provide the instructions for it and probably put it first, above option 2 (chmod)